index.js

```diff
@@ -47,12 +47,23 @@ EncryptedField.prototype.vault = function(name) {
     }
 };
 
-EncryptedField.prototype.field = function(name) {
-    var self = this;
+EncryptedField.prototype.field = function(name, config) {
+    var self            = this;
+    config              = config || {};
+    var hasValidations  = [undefined,null].indexOf(typeof config.validate) === -1;
+
+    config.validate  = config.validate || {};
+
+    var fieldType = (config.hasOwnProperty('type')) ? self.Sequelize.VIRTUAL(config.type) : self.Sequelize.VIRTUAL;
 
     return {
-        type: self.Sequelize.VIRTUAL,
+        type: fieldType,
```

Comment:
It's not that the field wouldn't be virtual. Virtual fields can optionally take a return type (http://docs.sequelizejs.com/en/latest/api/datatypes/#virtual)

So a generic virtual field would be:

```
Sequelize.VIRTUAL
```

But a virtual field that should return a boolean type would be:

```
Sequelize.VIRTUAL(Sequelize.BOOLEAN...
```

So the actual type will always be virtual, I'm just doing a check to see if the user explicitly included a return type. I will clean up my code and rebase off of master to include the last PR acceptance.